docs/releases.rst

```diff
@@ -1,6 +1,333 @@
+            version: 2
+          servers:
+            - address: "MailuServer:993"
+
+Security hardening
+^^^^^^^^^^^^^^^^^^
+
+We have gone further than ever. Now Mailu containers drop their privileges and communicate on separate networks. They also share the same base image where on x86 `a Hardened memory allocator <https://github.com/GrapheneOS/hardened_malloc>`_ is configured.
+
+Webmails which are running PHP make use of `Snuffleupagus <https://github.com/jvoisin/snuffleupagus>`_.
+
+
+New Functionality & Improvements
+````````````````````````````````
+
+For a list of all the changes (including bug fixes) refer to `CHANGELOG.md` in the root folder of the Mailu github project.
+
+A short summary of the new features:
+
+- Features: Provide auto-configuration files (autodiscover, autoconfig & mobileconfig); Please update your DNS records
+- Features: Introduction of the Mailu RESTful API. The full Mailu config can be changed via the Mailu API.
+  See the section Mailu RESTful API & the section configuration reference in the documentation for more information.
+- Features: Allow other folders to be synced by fetchmail
+- Features: Update the webmail images.
+  Roundcube
+
+    - Switch to base image (alpine)
+    - Switch to php-fpm
+
+  SnappyMail
+
+    - Switch to base image
+    - Upgrade php7 to php8.
+
+- Features: Implement Header authentication via external proxy
+- Features: Add FETCHMAIL_ENABLED to toggle the fetchmail functionality in the admin interface
+- Features: Create a polite and turtle delivery queue to accommodate destinations that expect emails to be sent slowly
+- Features: Add support for custom NGINX config in /etc/nginx/conf.d.
+- Features: Added ability to mark spam mails as read or unread when moving to junk folder.
+- Features: Switch from RainLoop to SnappyMail. SnappyMail has better performance and is more secure.
+- Features: Configurable default spam threshold used for new users
+- Features: Create a GUI for WILDCARD_SENDERS
+- Features: Prevent signups with accounts for which an SQL-LIKE alias exists.
+- Features: Introduce TLS_PERMISSIVE, a new advanced setting to harden cipher configuration on port 25. Changing the default is strongly discouraged, please read the documentation before doing so.
+- Features: Upgrade the anti-spoofing rule. We shouldn't assume that Mailu is the only MTA allowed to send emails on behalf of the domains it hosts... but we should also ensure that both the envelope from and header from are checked.
+- Features: Implement the required glue to make "doveadm -A" work
+- Features: Implement a minimum length for passwords of 8 characters. Check passwords upon login against HaveIBeenPwned and warn users if their passwords are compromised.
+- Features: Implement OLETools and block bad macros in office documents
+- Features: Switch to GrapheneOS's hardened_malloc
+- Features: New override system for Rspamd. In the old system, all files were placed in the Rspamd overrides folder.
```

Comment:
This is already covered in a point above, maybe just keep it as 1 line and reference the previous point in this document